### PR TITLE
Update to 0.15.1

### PR DESCRIPTION
**Pull request by brunocvcunha**
> **Pending Work**

Will concentrate my changes on this PR, so others don't need to redo the same work.

If someone wants to contribute based on my fork, ping me so I can set up the right permissions.

### CODE REVIEW

**Comment by brunocvcunha** [comment]
@ChristopherA @shannona Do you guys plan to add SegWit details? Anything specific that want to cover from 0.15.X?

File: 02_1_Setting_Up_a_Bitcoin-Core_VPS_by_Hand.md [hunk 1]
```diff
@@ -294,9 +294,9 @@ $ source ~/.bash_profile
 
 We suggest setting up two variables to make this installation more automatic.
 
-The first variable, $BITCOIN, should be set to the current version of Bitcoin. It was 0.14.1 when we wrote this. The second will then automatically generate a truncated form used by some of the files.
+The first variable, $BITCOIN, should be set to the current version of Bitcoin. It was 0.15.1 when we wrote this. The second will then automatically generate a truncated form used by some of the files.
 ```
-$ export BITCOIN=bitcoin-core-0.14.1
+$ export BITCOIN=bitcoin-core-0.15.1
```
Comment (shannona) [inline 02_1_Setting_Up_a_Bitcoin-Core_VPS_by_Hand.md]: @ChristopherA reported some problems with the StackScript initially when 0.15 came out. Were you able to confirm that it worked fine with your upgrades to 0.15.1?

**Comment by shannona** [comment]
And, yes, SegWit was in the table of contents at one time, but I think I cleared it because its adoption was so up in the air when we were writing regularly. I'd be entirely happy to see a Chapter 16 about SegWit. It'd be important to start it off with a great explanation of what SegWit is, before diving into how to use it.

**Comment by brunocvcunha** [comment]
Yes, I have followed the steps in the guide and got the testnet up and running on a Linode VPS.
Feel free to merge it, and I'll open another PR for the upcoming details & chapters.

**Pull request merged**